app/migrations/Version20171204142404.php

```diff
@@ -18,6 +22,13 @@ public function up(Schema $schema)
         $this->addSql('ALTER TABLE citizen_projects_skills ADD CONSTRAINT FK_B3D202D9EA64A9D0 FOREIGN KEY (citizen_project_skill_id) REFERENCES citizen_project_skills (id)');
     }
 
+    public function postUp(Schema $schema)
+    {
+        if ($this->container->get('app.migration.citizen_project_category_skill')) {
```

Comment:
Please use the method `has` http://api.symfony.com/3.4/Symfony/Component/DependencyInjection/ContainerInterface.html#method_has

Reply:
Oups! thx